real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/simulation/PlayBackUtil.java

```diff
@@ -9,6 +31,16 @@
 	  private static final String METADATA_NAME = "Robobuggy Data Logs";
 	  private static final String METADATA_SCHEMA_VERSION = "1.1";
 	  private static final String METADATA_HIGHLEVEL_SW_VERSION = "1.0.0";
+	  private static final  Publisher IMU_PUB = new Publisher(NodeChannel.IMU.getMsgPath());
```

Comment:
I dont feel very good about these being static, lets convert them to singleton-held